### ISSUE TITLE
[Defect] Editor previously able to publish events to specific sections may now only publish to national events

### ISSUE THREAD

**Issue by FranECross** [department-of-veterans-affairs/va.gov-cms#17919]
## Description

Reported in this Slack [thread](https://dsva.slack.com/archives/CDHBKAL9W/p1713482431921779), and associated to Jira ticket (which we don't have access to): https://va-gov.atlassian.net/browse/VAHELP-7044

From Ian Sears:
We have an Editor, properly assigned to the Outreach Hub, familiar with creating New Event and assigning it to a specific section (i.e. VSIN 9). However, recently their dropdown is limited to ONLY "national events".

It is possible this change may have taken place in late Jan or early Feb?

I can/could provide VSIN or even more granular section privileges. However this Editor has been making Outreach Hub Events for 10 months and only recently no longer sees section assignments in their event dropdown.

### User story

**AS AN** Outreach Hub editor with previous access to creating new events and assigning to specific sections i.e. VSIN 9
**I WANT** to be able to continue doing this, but am only now able to assign to National Events
**SO THAT** I can continue publishing events as usual

## Acceptance criteria

- [ ] Editor is again able to publish to specific sections, and isn't restricted to only national events

**Comment by ian-sears** [comment]
**Editor:** [andrew.ullom@va.gov](mailto:andrew.ullom@va.gov)
**Description:** I noticed when creating an event to list locally, there is no option to list local VAMC, the only option to list where the event will be listed is “Outreach and events: Outreach events”.
I noticed that I am no longer able to post locally to my VAMC events page. Is this the case for all VHA locations?
I was not able to assign a new post to the VHA Central Ohio Health Care location. It defaulted to the National Outreach Hub and did not have any other location options to choose from. I also recently noticed this happened back in November with 4 posts I made. One was listed locally but the other 3 were listed Nationally.
https://www.va.gov/central-ohio-health-care/events/?selectedOption=past
